src/ape/contracts/base.py

```diff
@@ -899,10 +900,18 @@ def receipt(self) -> ReceiptAPI:
                 return receipt
 
         # Brute force find the receipt.
-        receipt = self.chain_manager.contracts.get_creation_receipt(self.address)
+        receipt = self.meta.receipt
         self._cached_receipt = receipt
         return receipt
 
+    @cached_property
+    def meta(self):
```

Comment:
What is the return type here?

Reply:
`ContractCreation`, for some reason adding an annotation causes a circular import

Reply:
fixed in https://github.com/ApeWorX/ape/pull/2001/commits/65446305c255d94c7a416ec8913262b08f5d8f48